data/json/monsters/insect_spider.json

```diff
@@ -37,7 +37,7 @@
     "default_faction": "dark_wyrm",
     "bodytype": "snake",
     "species": [ "MUTANT" ],
-    "volume": "108500 ml",
```

Comment:
You cannot use decimals in unit field, see example
![image](https://github.com/user-attachments/assets/988c35f6-8371-4d27-9e10-87b3bbae6c4b)